package.json

```
@@ -1,68 +1,59 @@
 {
   "name": "featuretoggle",
-  "version": "0.1.0",
+  "version": "1.0.0",
   "description": "Feature Toggle admin tool and library for Enonic XP",
   "license": "Apache-2.0",
   "private": true,
   "scripts": {
-    "build:client": "webpack --config webpack.client.config.js --color --hide-modules",
-    "build:server": "webpack --config webpack.server.config.js --color --hide-modules",
+    "build:client": "webpack --config webpack.client.config.js --color",
+    "build:server": "webpack --config webpack.server.config.js --color",
     "build": "run-p -c build:*",
-    "dist:client": "cross-env NODE_ENV=production webpack --config webpack.client.config.js --color --hide-modules",
-    "dist:server": "cross-env NODE_ENV=production webpack --config webpack.server.config.js --color --hide-modules",
-    "dist": "run-p -c dist:*"
+    "dist:client": "cross-env NODE_ENV=production webpack --config webpack.client.config.js --color",
+    "dist:server": "cross-env NODE_ENV=production webpack --config webpack.server.config.js --color",
+    "dist": "run-p -c dist:*",
+    "build:react4xp": "npm explore @enonic/react4xp -- npm run build:react4xp"
   },
   "devDependencies": {
-    "@babel/core": "^7.5.4",
-    "@babel/plugin-proposal-class-properties": "^7.5.0",
-    "@babel/preset-env": "^7.5.4",
-    "@babel/register": "^7.4.0",
-    "@material-ui/core": "^4.11.3",
-    "@material-ui/icons": "^4.11.2",
-    "@material-ui/lab": "^4.0.0-alpha.57",
-    "@material-ui/styles": "^4.11.3",
-    "@reduxjs/toolkit": "^1.5.0",
-    "autoprefixer": "^9.6.1",
-    "axios": "^0.21.1",
-    "babel-loader": "^8.0.6",
-    "browserslist": "^4.6.6",
-    "browserslist-config-enonic": "^1.0.4",
-    "copy-webpack-plugin": "^5.0.3",
-    "cross-env": "^5.2.0",
-    "css-loader": "^2.1.1",
-    "cssnano": "^4.1.10",
-    "file-loader": "^3.0.1",
-    "glob": "^7.1.4",
-    "less": "^3.9.0",
-    "less-loader": "^4.1.0",
-    "mini-css-extract-plugin": "^0.6.0",
-    "node-sass": "^4.12.0",
+    "@emotion/styled": "^11.11.0",
+    "@enonic-types/lib-react4xp": "^4.0.0",
+    "@enonic/react4xp": "4.1.0",
+    "@mui/material": "^5.14.7",
+    "@reduxjs/toolkit": "^1.9.5",
+    "@types/react": "^18.2.21",
+    "autoprefixer": "^10.4.15",
+    "axios": "^1.5.0",
+    "babel-loader": "^9.1.3",
+    "browserslist": "^4.21.10",
+    "browserslist-config-enonic": "^1.0.8",
+    "cross-env": "^7.0.3",
+    "css-loader": "^6.8.1",
+    "cssnano": "^6.0.1",
+    "file-loader": "^6.2.0",
+    "glob": "^10.3.3",
+    "mini-css-extract-plugin": "^2.7.6",
+    "node-sass": "^9.0.0",
```

`node-sass` has been deprecated, and I'm not able to build this project on Linux. Can you replace it with the `sass` package?

```suggestion
    "sass": "^1.67.0",
```